src/pages/GamePage.js

```diff
@@ -9,4 +11,18 @@ class GamePage extends Component {
   }
 }
 
-export default GamePage;
+const mapStateToProps = ({ gameSettings: { difficulty, cattegory, type } }) => ({
+  configs: {
+    difficulty,
+    cattegory,
+    type,
+  },
+});
+
+GamePage.propTypes = {
+  difficulty: PropTypes.string,
+  cattegory: PropTypes.string,
+  type: PropTypes.string,
+}.isRequire;
```

Comment:
Não seria da forma que está abaixo? :rocket: 

```suggestion
}.isRequired;
```